napari/components/_layer_slicer.py

```diff
@@ -102,7 +142,7 @@ def slice_layers_async(
         # when we want to perform sync slicing anyway.
         requests = {}
         for layer in layers:
-            if isinstance(layer, _AsyncSliceable):
+            if isinstance(layer, _AsyncSliceable) and not self._force_sync:
                 requests[layer] = layer._make_slice_request(dims)
             else:
                 layer._slice_dims(dims.point, dims.ndisplay, dims.order)
```

Comment:
The other thing where I'm curious for feedback from others is if `force_sync` should wait for the task submitted to the other thread. Or if it should use `_slice_dims`. We don't have to address that here - it could be in one of the integration tasks.

Reply:
If I'm following you correctly, we had also discussed adding 
```
if self.force_sync:
  task.result()
```
to ensure the wait. This would go at line 151 (just after the line submitted the task).

Reply:
Yeah, no need to address now. But it would be good to get more perspective/feedback on this as it goes through further review.